src/components/Settings/DevicesTab/index.js

```diff
@@ -218,7 +218,25 @@ function mapDispatchToProps(dispatch) {
   };
 }
 
+const LoadingContainer = props => (
```

Comment:
The Loading container should be visually similar to other Loaders with a Spinner in the center.

Reply:
@akshatnitd This is not the circular loading container you are talking about.This loading container is by default coming from the googleapiwrapper(only for the devices tab) after the circular loading container in settings.react.js. As I could not do any changes in the apiwrapper that's why I made changes exclusively.

Reply:
You can see my PR #2360 where I have put the screenshots. Actually I made this new PR because of some issue..